Assets/MRTK/Core/Inspectors/Profiles/MixedRealitySpatialAwarenessMeshObserverProfileInspector.cs

```diff
@@ -1,5 +1,5 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.﻿
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Slight nit, but we standardized the license headers to the current Microsoft recommendation in #8102, but it looks like this PR is reverting them in almost every file it touches. Might be best to revert these header changes.